packages/client/vite.config.js

```diff
@@ -15,6 +15,7 @@ import tsconfigPaths from 'vite-tsconfig-paths'
 
 process.env.VITE_APP_COUNTRY_CONFIG_URL =
   process.env.COUNTRY_CONFIG_URL || 'http://localhost:3040'
+process.env.VITE_APP_VERSION = process.env.VITE_APP_VERSION || 'Development'
```

Comment:
`process.env.VITE_APP_VERSION = process.env.VERSION || 'Development'`
I think this one should be like this to actually use the VERSION environment variable and to add a default for it. See the environment variable on top of it